recipe/meta.yaml

```
@@ -48,19 +53,22 @@ requirements:
     - make                          # [unix]
     - m2-make                       # [win]
     - {{ compiler('c') }}           # [unix]
-    - {{ stdlib('c') }}             # [unix]
     - {{ compiler('m2w64_c') }}     # [win]
-    - {{ stdlib('m2w64_c') }}       # [win]
     - autoconf                      # [unix]
     - automake                      # [unix]
     - gettext                       # [unix]
     - libtool                       # [unix]
     - sed                           # [unix]
   host:
-    - xorg-xorgproto
     - libxcb
     - pthread-stubs
+    - xorg-xextproto
+    - xorg-kbproto
+    - xorg-inputproto
+    - xorg-libxdmcp
+    - xorg-libxau
     - xorg-util-macros
+    - xorg-xorgproto
     - xorg-xtrans
```

```suggestion
    - libxcb
    - pthread-stubs
    - xorg-xextproto
    - xorg-kbproto
    - xorg-inputproto
    - xorg-util-macros
    - xorg-xorgproto
    - xorg-xtrans
```
Wrong version of `libxcb` again.

The miscellaneous proto packages don't seem to be needed for the conda-forge build of `libx11` but I do see them mentioned in the logs for our failed builds. Can't hurt to keep them since we now have these as packages but I'm not sure they weren't for some optional target we could have skipped building.

---

Fixed